app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= turbo_frame_tag "modal" do %>
+  <div class="modal">
```

Comment:
:red_circle: Kindly increase the background opacity of this block. The transparency is somehow annoying. 
You can for example use this :arrow_down: 
`<div class="modal" style="background-color:rgba(0, 0, 0, 0.8)"> `
As this change will darken the background You might find it hard to read the heading text due to its black color. So my advice would be to make it white as follow :arrow_right:    ` <h1 class="text-white">Generate Shopping List</h1>`